tests/functional/z_cluster/nodes/test_worker_nodes_network_failures.py

```diff
@@ -48,7 +48,7 @@ def setup(self, request, interface, multi_pvc_factory, deployment_pod_factory):
             interface(str): The type of the interface
                 (e.g. CephBlockPool, CephFileSystem)
             multi_pvc_factory: A fixture create a set of new PVCs
-            dc_pod_factory: A fixture to create dc pod
+            deployment_pod_factory: A fixture to create dc pod
```

Comment:
Why not make the comment is - "A fixture to create deployment pod"

Here "dc" is -  deploymentConfig or Deployment

Reply:
That is right, updated the docstring.